MAUI/Cartesian-Charts/BoxAndWhisker.md

```diff
@@ -0,0 +1,340 @@
+---
+layout: post
+title: Box and whisker Chart in .NET MAUI Chart control | Syncfusion
+description: Learn here all about BoxAndWhisker chart support in Syncfusion .NET MAUI Chart (SfCartesianChart) control.
```

Comment:
At all place mention BoxAndWhisker  => Box and Whisker, mentioning API only use like above.

Reply:
Okay. I have replaced